widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.kt

```diff
@@ -804,7 +804,7 @@ internal class ChatController(
         currentChatItems: MutableList<ChatItem>,
         message: ChatMessage
     ) {
-        if (!message.content.isNullOrEmpty()) {
+        if (message.content.isNotEmpty()) {
```

Comment:
Are these significant changes? (Making fields nullable or not nullable)

Reply:
I think not, because before that they have no strict nullability in Java. Fields mainly turned into the safer/non-nullable type so it is possible just to use them as they are before.

Reply:
Hm, I'm also not sure if the change from `isNullOrEmpty()` to just `isNotEmpty()` won't affect anything.
It's okay if you are completely sure.

Reply:
@DavDo Cool, thank you for the explanation. I understand now and agree with the change. 👍